substrate/frame/treasury/src/lib.rs

```diff
@@ -106,7 +106,7 @@ use frame_support::{
 	weights::Weight,
 	BoundedVec, PalletId,
 };
-use frame_system::pallet_prelude::BlockNumberFor;
+use frame_system::pallet_prelude::BlockNumberFor as SystemBlockNumberFor;
```

Comment:
Related to #6245, someday this type alias can be removed to avoid confusion.

Reply:
May be local block number? What system really means here. Also `BlockNumberFor` reference to the block's header, not system pallet